modules/Bio/EnsEMBL/Variation/VariationFeature.pm

```diff
@@ -1968,7 +1967,84 @@ sub hgvs_genomic {
 
 }
 
+=head2  spdi_genomic
+
+  Description : Returns a reference to a hash with the allele as key and a string with the genomic SPDI notation of this VariationFeature as value. By default uses the
+                slice it is placed on as reference. 
+  Returntype  : Hash reference
+  Exceptions  : Throws exception if VariationFeature can not be described relative to the feature_Slice of the supplied reference feature
+  Caller      : general
+  Status      : Experimental
+
+=cut
+
+sub spdi_genomic{ 
+
+  my $self = shift; 
+  
+  my $ref_feature = $self;   
+  my %spdi; 
+
+  my $ref_slice = $self->slice; 
```

Comment:
As VariationFeatures can be created without slices, it would be better to check we have something here and throw if not.